src/controller/news.controller.ts

```diff
@@ -0,0 +1,71 @@
+import { Request, Response } from 'express';
+import { APIResponse } from "../../model/APIResponse";
+import multer, { FileFilterCallback } from "multer";
+import path from "path";
+import { MIMETYPES } from "./mimeTypes";
+import fs from "fs";
+
+class NewsController {
+
+    private static storagePath = 'fileStash/news/'
+
+    private static multerStorage = multer.diskStorage({
+        destination: (req: Request, file: Express.Multer.File, cb) => {
+            cb(null, NewsController.storagePath);
+        },
+        filename: (req: Request, file: Express.Multer.File, cb) => {
+            cb(null, file.fieldname + '_' + Date.now() + path.extname(file.originalname));
+        }
+    });
+
+    private static multerLimits = {
+        fileSize: 1 * 1024 * 1024,
```

Comment:
why 1024^2?
why are limits set to 5 files, 10 parts?

Reply:
I will remove the whole multer setup as it is not nessecary if we will never allow user uploads in this context

Reply:
LGTM!